package.json

```diff
@@ -12,11 +12,28 @@
   "files": [
     "src"
   ],
+  "exports": {
+    ".": {
+      "types": "./src/schema.d.ts",
+      "default": "./src/schema.d.ts"
+    },
+    "./zod": {
+      "import": "./src/schema.zod.ts",
+      "default": "./src/schema.zod.ts"
+    }
+  },
   "scripts": {
-    "generate": "npx openapi-typescript https://raw.githubusercontent.com/modelcontextprotocol/registry/refs/tags/v1.2.3/docs/reference/api/openapi.yaml -o src/schema.d.ts --export-type"
+    "generate": "npm run generate:types && npm run generate:zod",
+    "generate:types": "npx openapi-typescript https://raw.githubusercontent.com/modelcontextprotocol/registry/refs/tags/v1.2.3/docs/reference/api/openapi.yaml -o src/schema.d.ts --export-type",
+    "generate:zod": "npx openapi-zod-client https://raw.githubusercontent.com/modelcontextprotocol/registry/refs/tags/v1.2.3/docs/reference/api/openapi.yaml -o src/schema.zod.ts --export-types --export-schemas"
   },
   "devDependencies": {
     "openapi-typescript": "^7.8.0"
   },
-  "packageManager": "pnpm@10.17.1+sha512.17c560fca4867ae9473a3899ad84a88334914f379be46d455cbf92e5cf4b39d34985d452d2583baf19967fa76cb5c17bc9e245529d0b98745721aa7200ecaf7a"
+  "packageManager": "pnpm@10.17.1+sha512.17c560fca4867ae9473a3899ad84a88334914f379be46d455cbf92e5cf4b39d34985d452d2583baf19967fa76cb5c17bc9e245529d0b98745721aa7200ecaf7a",
+  "dependencies": {
```

Comment:
suggestion(non-blocking): put these new dependencies with the existing one in "devDependencies" section